ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/BatchingParallelIngestionPipe.scala

```diff
@@ -20,8 +20,10 @@ object BatchingParallelIngestionPipe {
       batcher: IN_BATCH => Future[DB_BATCH],
       ingestingParallelism: Int,
       ingester: DB_BATCH => Future[DB_BATCH],
-      tailer: (DB_BATCH, DB_BATCH) => DB_BATCH,
-      ingestTail: DB_BATCH => Future[DB_BATCH],
+      maxTailerBatchSize: Int,
+      tailerSeed: DB_BATCH => Vector[DB_BATCH],
```

Comment:
This tailer seed can be fairly trivial if we are zeroing out DB payload already earlier.
How about we zero them out inside the ingester, and then we can remove this injection?

Reply:
Done